lib/ronin/db/web_vuln.rb

```diff
@@ -0,0 +1,202 @@
+# frozen_string_literal: true
+#
+# ronin-db-activerecord - ActiveRecord backend for the Ronin Database.
+#
+# Copyright (c) 2022-2023 Hal Brodigan (postmodern.mod3 at gmail.com)
+#
+# ronin-db-activerecord is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ronin-db-activerecord is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with ronin-db-activerecord.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+require 'ronin/db/model'
+require 'active_record'
+
+module Ronin
+  module DB
+    #
+    # Represents parsed URLs.
+    #
+    class WebVuln < ActiveRecord::Base
+
+      include Model
+
+      # @!attribute [rw] id
+      #   The primary key of the URL.
+      #
+      #   @return [Integer]
+      attribute :id, :integer
+
+      # @!attribute [rw] scheme
+      #   The scheme of the URL.
+      #
+      #   @return [URL]
+      belongs_to :url, required:   true,
+                       class_name: 'URL'
+
+      # @!attribute [rw] type
+      #   The type of vuln.
+      #
+      #   @return [String, nil]
+      enum type: {
+        lfi:           'lfi',
+        rfi:           'rfi',
+        sqli:          'sqli',
+        ssti:          'ssti',
+        open_redirect: 'open_redirect',
+        reflected_xss: 'reflected_xss'
+      }
+
```

Comment:
I think we need `validates :type, presence: true` here to ensure the `type` enum is set?